.idea/gradle.xml

```diff
@@ -20,7 +20,8 @@
           </set>
         </option>
         <option name="resolveModulePerSourceSet" value="false" />
+        <option name="useQualifiedModuleNames" value="true" />
```

Comment:
Please don't include unrelated config changes here.

Also, reminder to make sure file changes include EOFs (below).

Reply:
ah, committed by mistake as update the AS.